gogi.go

```diff
@@ -9,7 +9,7 @@ import (
 const (
 	version       = "0.0.4"
 	ua            = "gogi/" + version
-	defaultAPIURL = "https://www.gitignore.io"
+	defaultAPIURL = "https://www.toptal.com/developers/gitignore"
```

Comment:
You should just change the `defaultAPIURL` to `https://www.toptal.com`, then change the `typePath` from `/api` to `/developers/gitignore/api`.